### ISSUE TITLE
Youtube non-authenticated user 

### ISSUE THREAD

**Issue by ReginaG6** [manikandanraji/youtubeclone-frontend#20]
Hello, 

I was wondering how we can allow a user to view certain pages like the home page/trending etc. without signing in, much more like YouTube. Currently, the user needs to sign in to view anything.

**Comment by manikandanraji** [comment]
Yep, it's kind of annoying. For that, two changes need to be made. One in the backend where we need to make certain routes public. In the frontend, you need to limit the user from liking, commenting, subscribing to a channel when the user is not logged in. I'll look into this.

**Comment by ReginaG6** [comment]
No worries
